biojava-aa-prop/src/main/java/org/biojava/nbio/aaproperties/Utils.java

```diff
@@ -68,10 +69,10 @@ public final static double roundToDecimals(double d, int c) {
 	 * 		true if invalid characters are found, else return false.
 	 */
 	public final static boolean doesSequenceContainInvalidChar(String sequence, Set<Character> cSet){
-		for(char c:sequence.toCharArray()){
-			if(!cSet.contains(c)) return true;
-		}
-		return false;
+			boolean result = sequence.chars().filter(c -> !cSet.contains((char) c))
+																 .findFirst()
+																 .isPresent();
+			return result;
```

Comment:
I'd say the previous implementation is more efficient and I'd argue also more readable